include/swift/AST/DiagnosticsSema.def

```diff
@@ -2607,18 +2607,18 @@ NOTE(add_self_to_type,none,
 
 WARNING(warn_unqualified_access,none,
         "use of %0 treated as a reference to %1 in %2 %3",
-        (Identifier, DescriptiveDeclKind, DescriptiveDeclKind, DeclName))
+        (DeclBaseName, DescriptiveDeclKind, DescriptiveDeclKind, DeclName))
```

Comment:
This one probably should be an Identifier, since it comes from something the user wrote. Yes, no?